src/Archive.ts

```diff
@@ -156,10 +161,29 @@ class Board {
   }
 
   prepForSave() {
-    this._systems = Object.fromEntries(this.Systems)
+    this._systems = Object.fromEntries(this.systems)
   }
 }
 
+interface SaveFile {
+  players: Player[]
+  board: {
+    systems: Systems
+  }
+  /**
+   * @param scrap - Array of card objects scrapped from the game (implied cards from fate history are excluded)
+   */
+  scrap: Card[]
+  addedFaithfulActionCards?: Card[]
+  idealMarkers?: string[]
+  act: 1 | 2 | 3
```

Comment:
No Act 1 save file since this only supports intermissions

Reply:
Done.